goon/browserassets/css/browserOutput-dark.css

```
@@ -439,3 +446,33 @@ span.body .coderesponses 		{color: #FF3333;}
 .announcement.sec h1 { color: #f00; font-size: 180%; font-family: Verdana, sans-serif; }
 
 .bolditalics {font-style: italic; font-weight: bold;}
+
+.boxed_message {
+	text-align: center;
+	border-style: solid;
+	border-width: medium;
+	border-color: #a4bad6;
+	border-radius: 5px;
+	margin: 25px;
+	padding: 15px;
+  }
```

```suggestion
 }
```
This is a more of a nitpick than anything, but there are a few rules in this PR that have an extra space with the ending bracket. Can we remove this extra space from those to match the formatting with the already existing rules in the PR?